### PR TITLE
Contact forms fix

### PR DESCRIPTION
**Pull request by glavery**

### CODE REVIEW

File: dit_helpdesk/contact/constants.py [hunk 1]
```diff
@@ -5,3 +5,4 @@
 DDAT_SUBJECT_SUFFIX = " - DDAT Support Team"
 FEEDBACK_EMAIL = ""
 FEEDBACK_FULLNAME = ""
+HMRC_TAX_FORM_URL = "https://www.tax.service.gov.uk/shortforms/form/CITEX_CGEF"
```
Comment (trbateman) [inline dit_helpdesk/contact/constants.py]: Does this need to be hardcoded?

File: dit_helpdesk/contact/views.py [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
 from django.conf import settings
+from django.contrib import messages
 from django.http import HttpResponseRedirect
-from django.shortcuts import render_to_response
+from django.shortcuts import render_to_response, redirect
 from django.template.loader import get_template
+from django.urls import reverse
 from formtools.wizard.views import SessionWizardView
 
 from contact import constants
```
Comment (trbateman) [inline dit_helpdesk/contact/views.py]: Can these variables go into settings.py? Developers / Ops might not be expecting to find settings in anything other than the usual settings.py [base.py, development.py, production.py etc].

File: dit_helpdesk/contact/constants.py (hunk 1 above)
Comment (glavery) [reply dit_helpdesk/contact/constants.py]: done

File: dit_helpdesk/contact/views.py (hunk 2 above)
Comment (glavery) [reply dit_helpdesk/contact/views.py]: done. moved into settings base.py
Although there is instances elsewhere where a constants.py has been used, forms api for example, which is where i got the idea. 
the idea being that non sensitive stuff is defined and and stored as part of the app, and sensitive stuff is in settings and available via the env variables.

**Pull request merged**